tools/check-symbols.sh

```
@@ -0,0 +1,23 @@
+
+set -eu
+set -o pipefail
+
+trap 'echo "🔴 FAILED"' EXIT
+
+if [ $# -eq 0 ]; then
+    echo "No ELF files specified"
+    exit 1
+fi
+
+for ELF in $*
```

Shellcheck (0.11.0)

[warning] Use "$@" (with quotes) to prevent whitespace problems.

(SC2048)